### PR TITLE
Add columns to nearby_competitions_table

### LINKED ISSUE #2482: Two new columns in _nearby_competitions_table.html.erb for the board
The quality of the requested competitions could be better checked if the board (or any team announcing in the future) could have two new columns in this table: "limit" and "number of competitors" (the latter only if results are posted).

### PR DESCRIPTION
**Pull request by pedrosino**
Fixes #2482 

@Luis-J-Ianez I added "---" in case the competition has no limit assigned. Let me know if you prefer just empty, or if you prefer that I add something to the Competitors column if the competition is not over yet.

Also, I aligned the competitor count to the center, since it looked a little off when aligned to the left (see below).

center aligned
image

left aligned
image

EDIT

Another screenshot, this time from the competitions withn 365 days
image

### CODE REVIEW

**Comment by pedrosino** [comment]
It was not the topic here, but I found the modal window too small, given it appears "alone" in the screen.

Compare this following screenshot with the one just above:
image


Now a smaller window:
90% width
image

800px width
image)


As you can see, the difference is not much in small screen, but it looks much better in wider ones.
I made another commit changing this. @jfly and @jonatanklosko let me know if this is ok.

**Comment by Luis-J-Ianez** [comment]
@pedrosino, thanks for doing this. Even if I'm not very concerned about aesthetic topics, as you're asking, I'd say blank is better for both columns when the results are not up or there's no limit, centered in both cases is better too, and definitely a wider modal screen would be appreciated. Thanks!

File: WcaOnRails/app/views/competitions/_nearby_competitions_table.html.erb [hunk 1]
```diff
@@ -27,6 +29,8 @@
       <td class="text-nowrap text-right">
         <%= link_to_google_maps_dir "#{@competition.kilometers_to(c).round(2)} km", c.latitude_degrees, c.longitude_degrees, @competition.latitude_degrees, @competition.longitude_degrees %>
       </td>
+      <td><%= c.competitor_limit_enabled ? c.competitor_limit : "---" %></td>
+      <td style="text-align: center;"><%= c.competitors.count if c.is_probably_over? %></td>
```
Comment (jfly) [inline WcaOnRails/app/views/competitions/_nearby_competitions_table.html.erb]: Nit: I think we tend to use `class="text-center"` for stuff like this. Slightly shorter, and avoids using inline styles, which can be controversial.

Comment (pedrosino) [reply WcaOnRails/app/views/competitions/_nearby_competitions_table.html.erb]: I'll use that one, thanks.

**Pull request merged**